packages/core/src/routes/+page.svelte

```diff
@@ -84,8 +85,31 @@ let buttonClickedTimes = 0;
   />
 </Label>
 
-<!-- Pill -->
+<!-- Switch -->
+<Switch
+  enabled
+  variant="annotated"
+/>
+
+<Switch
+  enabled
+  label="Lunchtime"
+/>
 
+<Switch
+  enabled
```

Comment:
This is a great callout - I think I like `checked` the most of those since that is consistent with `<input type="<checkbox|radio>" />`. My next choice is `on`. I do not like string `'on' | 'off'` as options for a boolean value.

Reply:
Let's go with `on` since it seems to be the best compromise, there's something about mapping checked on to a switch that is breaking my brain 🤣 😭